Sources/Algorithms/Chunked.swift

```diff
@@ -218,7 +221,11 @@ extension LazySequenceProtocol where Self: Collection, Elements: Collection {
   }
   
   /// Returns a lazy collection of subsequences of this collection, chunked by
-  /// grouping elements that project to the same value.
+  /// grouping elements that project to the equal values.
```

Comment:
`to the equal values.` Seems not to read correctly, would that be `to equal values`?

Reply:
```suggestion
  /// grouping elements that project to equal values.
```